lra/certifications/microprofile-lra-1.0/CCR-results.html.haml

```diff
@@ -0,0 +1,275 @@
+
+%p
+  The only difference between LRA 1.0 and LRA 1.0-RC2 is a small README file update -
+  %a{:href => "https://github.com/eclipse/microprofile-lra/commit/932f4a631a482c059591b9e0a1b3b509a6d96407"} https://github.com/eclipse/microprofile-lra/commit/932f4a631a482c059591b9e0a1b3b509a6d96407
+  which is not influencing the TCK results. The API and TCK binaries are the same.
+
+.h3 Implementation details
+
+%ul
+  %li
+    %p
+      Implementation Name -
+      %a{:href => "https://github.com/jbosstm/narayana"} Narayana
+  %li
+    %p
+      Version - 5.11.0.Final
+  %li
+    %p
+      Download Binary -
+      %a{:href => "http://www.jboss.org/jbosstm/downloads/5.11.0.Final/binary/narayana-full-5.11.0.Final-bin.zip"} http://www.jboss.org/jbosstm/downloads/5.11.0.Final/binary/narayana-full-5.11.0.Final-bin.zip
+  %li
+    %p
+      Download Source Code -
+      %a{:href => "http://www.jboss.org/jbosstm/downloads/5.11.0.Final/src/narayana-full-5.11.0.Final-src.zip"} http://www.jboss.org/jbosstm/downloads/5.11.0.Final/src/narayana-full-5.11.0.Final-src.zip
+  %li
+    %p
+      VCS revision tag
+      %a{:href => "https://github.com/jbosstm/narayana/tree/5.11.0.Final"} https://github.com/jbosstm/narayana/tree/5.11.0.Final
+  %li
+    %p
+      VCS revision commit
+      %a{:href => "https://github.com/jbosstm/narayana/commit/7c964826687a0619e91d63d633936942e6a3a34c"} https://github.com/jbosstm/narayana/commit/7c964826687a0619e91d63d633936942e6a3a34c
+
+.h3 Compatibility Certification Request JDK 11
+
+.h4 Environment details
+
+%ul
+  %li
+    %p
+      %strong Output of `uname -a`
+      %pre.language-haml
+        %code
+          :preserve
+            Linux XYZ 5.10.15-200.fc33.x86_64 #1 SMP Wed Feb 10 17:46:55 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
+  %li
+    %p
+      %strong Output of `java -version`
+      %pre.language-haml
+        %code
+          :preserve
+            openjdk version "11.0.8" 2020-07-14
+            OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.8+10)
+            OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.8+10, mixed mode)
+  %li
+    %p
+      %strong Output of `mvn -version`
+      %pre.language-haml
+        %code
+          :preserve
+            Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
+            Maven home: /opt/maven/apache-maven-3.6.3
+            Java version: 11.0.8, vendor: AdoptOpenJDK, runtime: /opt/java/hotspot/jdk-11.0.8+10
+            Default locale: en_US, platform encoding: UTF-8
+            OS name: "linux", version: "5.10.15-200.fc33.x86_64", arch: "amd64", family: "unix"
+
+.h4 CCR execution JDK 11
+
+%ul
+  %li
+    %p
+      Command -
+      %code WORKSPACE=$PWD PROFILE=LRA ./scripts/hudson/narayana.sh
+  %li
+    %p{:class => page.output_path.start_with?("/lra") ? "active": ""}
+      Results failsafe-reports zip -
+      %a{:href => relative("/lra/certifications/microprofile-lra-1.0/lra-1.0-jdk-11-ccr-results.zip")} lra-1.0-jdk-11-ccr-results.zip
```

Comment:
I don't know haml but we have a similar link to files which appear to have a different syntax. Eg

```
                <li class="list-group-item">
                  <a href="..//docs/specs/server-integration-guide.pdf">
                    <i class="icon-external-link"></i>
                    Narayana Server Integration Guide, v4.14.0, 2010
                  </a>
                </li>
```

I was unable to check it on my laptop because installing awestruct on my laptop fails badly.

If you are unable to install awestruct as well then the quickest way to verify the syntax would be to tierh
a) merge this PR and then we check. If it's wrong then we just raise another PR.
b) ask Ondra if he would be able to check out your PR and build it locally.

WDYT?

There is also file called `./_config/links.yml` which contains file links. I don't know what it's for or whether you need it for these two zips.

Reply:
yes, awestruck was paint to run but I was successful and I am running it locally and verifying everything that I push with awestruck -d. I'll check if this should be updated but the current version works for sure.

Reply:
@mmusgrov I wasn't able to do the relative links. All links even in the relative() directive/method need to start with / so they need to be absolute. The linked example also resolves the paths dynamically from the root /. I don't think that this is such a big issue as this path shouldn't change nevertheless as it will be linked in the official CCR.

Reply:
Okay I'll approve it and merge. Thanks for this.